### ISSUE TITLE
drop `monthly_pick` table

### ISSUE THREAD

**Issue by EnTeQuAk** [mozilla/addons-server#11843]
After https://github.com/mozilla/addons-server/pull/11840/ we should be able to drop the actual database table for the `monthly_pick` table.

Let's do this in the next tag or so.

**Comment by Abhi-khandelwal** [comment]
@EnTeQuAk , Can I take this one also :)

**Comment by EnTeQuAk** [comment]
Absolutely! I'll assign it to you right away if that's alright. Let us know if you need any help.

**Comment by Abhi-khandelwal** [comment]
To fix this , I guess i need to make changes in migrations files, Am I right ?

**Comment by EnTeQuAk** [comment]
That's correct, you need to create a new file in `migrations` that uses an increased number and in there you're going to have to write the SQL for dropping the table. There are quite a few examples in that folder as well for inspiration.
